src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java/JavaLanguageLexer.g4

```diff
@@ -168,7 +168,6 @@ LITERAL_CASE:            'case';
 LITERAL_CATCH:           'catch';
 LITERAL_CHAR:            'char';
 LITERAL_CLASS:           'class';
-LITERAL_CONST:           'const';
```

Comment:
These are some leftover artifacts that we don't need. The original thinking was that Java might implement these keywords one day, and to have them here for some reason. They aren't doing us any good, and aren't used at all.

Reply:
See here for details: https://docs.oracle.com/javase/specs/jls/se21/html/jls-3.html#jls-3.9:~:text=The%20keywords%20const%20and%20goto%20are%20reserved%2C%20even%20though%20they%20are%20not%20currently%20used